proto/ibc/core/client/v1/tx.proto

```diff
@@ -90,6 +94,28 @@ message MsgUpgradeClient {
 // MsgUpgradeClientResponse defines the Msg/UpgradeClient response type.
 message MsgUpgradeClientResponse {}
 
+// MsgScheduleIBCClientUpgrade defines an sdk.Msg to schedule an upgrade of an IBC client using a v1 governance proposal
+message MsgScheduleIBCClientUpgrade {
```

Comment:
what about `MsgScheduleIBCUpgrade`? The upgrade provided here is to upgrade the chain, but it includes information for the counterparty to upgrade their client. The msg is also defined in clienttypes, so I feel `clienttypes.MsgScheduleIBCUpgrade` sounds nice as opposed to `clienttypes.MsgScheduleIBCClientUpgrade`, feels a bit redundant

Reply:
open to other names as well, `MsgScheduleChainUpgrade` could work

Reply:
We can mimic the [`MsgSoftwareUpgrade`](https://github.com/cosmos/cosmos-sdk/blob/main/x/upgrade/types/tx.pb.go#L37)?

Reply:
sure, sounds good - drop the "client" since we're in the client package 👍

Reply:
I'm totally fine with `IBCSoftwareUpgrade` if everyone is fine with that! mimicking the SDK